packages/docs/docs/advanced/canvas-operations.mdx

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+slug: /canvas-operations
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Canvas Operations
+
+Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, we
+can make use several canvas operations that are provided by the Browser.
+
+## Filters
+
+<AnimationPlayer name="canvas-operations-filters" />
+
+Every Component accepts a list of filters. You can find a list of all filters
+the Browser can understand on
+[MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/filter).
+
+All filters available in Motion Canvas can be found under
+`@motion-canvas/2d/lib/partials`
+([API Documentation](/api/2d/partials#Functions)). Note that some filters, like
+`opacity`, can be set directly on the [`Node`](/api/2d/components/Node#opacity),
+and as a result are not exported from [partials](/api/2d/partials#Functions).
+
+`filters` accepts an array. This means that you can stack multiple filters onto
```

Comment:
Signals for all filters can also be accessed using properties:
```ts
yield* node().filters.blur(20, 1);
```
In this case, `filters` will try to find the first `blur` in the array, and if it can't do that it will create and add a new one to the list.

Reply:
So something like 
```ts
yield* node().filters.blur(20, 1);
yield* node().filters.saturation(0, 1);
```
would be equivalent to 
```tsx
const blurValue = createSignal(0);
const saturationValue = createSignal(1);

yield scene.add(<Node filters={[blur(blurValue), saturation(saturationValue)]} />)
yield* blurValue(20, 1);
yield* saturationValue(0, 1);
```
?

That might actually be a more preferable syntax :D

Reply:
Yes, exactly